web/custom-plugins/forecast-of-response/src/utils/insertDocument.ts

```diff
@@ -0,0 +1,29 @@
+import { DmssAPI } from '@dmt/common'
+
+const DEFAULT_DATASOURCE_ID = 'ForecastDS'
+const DEFAULT_DIRECTORY = 'ForecastOfResponse'
+
+export const insertDocument = (
```

Comment:
rename this to addToPath? So it's less confusion in what it does in regards to the DMSS api